src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Constructors.jl

```diff
@@ -75,11 +92,13 @@ projective_space(
                 ) where {CoeffRingType<:Ring} = projective_space(A, Symbol.(var_names))
 
 
-function projective_space(
-    A::CoeffRingType, 
-    r::Int; 
-    var_name::String="s"
-  ) where {CoeffRingType<:Ring}
+@doc raw"""
+    projective_space(A::Ring, r::Int; var_name::String="s")
+
+Create the (relative) projective space `Proj(A[s₀,…,sᵣ])` over `A` 
+where `s` is a string for the variable names.  
+"""
+function projective_space(A::Ring, r::Int; var_name::String="s")
   R, _ = polynomial_ring(A, [var_name*"$i" for i in 0:r])
```

Comment:
For a future PR (similar in other places).
```suggestion
function projective_space(A::Ring, r::Int; var_name::VarName=:s)
  R, _ = polynomial_ring(A, [Symbol(var_name, i) for i in 0:r])
```